api/toko-rest-api.yaml

```diff
@@ -117,6 +117,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/User'
+
+  /users/{userId}:
+    delete:
+      tags:
+        - Users
+      summary: Delete user by ID
+      description: For valid response try integer IDs with positive integer value.         Negative
+        or non-integer values will generate API errors
+      operationId: deleteUser
+      parameters:
+        - name: userId
+          in: path
+          description: ID of the user that needs to be deleted
+          required: true
+          schema:
+            minimum: 1.0
```

Comment:
Why do we need to define the minimum value for ID? And why is it decimal and not integer?